tests/circuit_graph/test_qasm.py

```diff
@@ -177,15 +177,13 @@ def test_state_initialization_decomposition(self):
             include "qelib1.inc";
             qreg q[2];
             creg c[2];
+            ry(1.5707963267948968) q[0];
             ry(1.5707963267948968) q[1];
-            ry(1.5707963267948963) q[0];

```

Comment:
There are three different changes in these qasm tests: 
a) The last digit above was adjusted, since the changes seem to give rise to a minimal difference at numerical precision. The reason why I took the liberty to just overwrite is that the new computations are more consistent, because pi/2 should lead to the same number in both RY gates (which was previously not given).
b) The wire reordering (instead of state reshape) requires all wires to be permuted.
c) The if condition avoiding rotations with a zero angle means that some gates in the queue do not appear any more.